lang/nl.js

```diff
@@ -1,55 +1,59 @@
+	"components.form-error-summary.errorSummary": "$$$ {count, plural, one {$$$ 1 $$$$$} other {$$$$$ {count} $$$$$$$$}} $$$$$$$$$$ $$$ $$$ $$$$$$$$$$$$$ $$$$ $ $$$$$ $$$$$$$$$$$$",
+	"components.input-date-range.endDate": "$$$$$$$$$$$$",
+	"components.input-date-range.errorBadInput": "{startLabel} $$$$$ $$$$$ {endLabel} $$$$$$$$",
+	"components.input-date-range.startDate": "$$$$$$$$$$$$$",
+	"components.input-date-time-range-to.to": "$$$$",
+	"components.input-date-time-range.endDate": "$$$$$$$$$$$$",
+	"components.input-date-time-range.errorBadInput": "{startLabel} $$$$$ $$$$$ {endLabel} $$$$$$$$",
+	"components.input-date-time-range.startDate": "$$$$$$$$$$$$$",
+	"components.input-date-time.date": "$$$$$$$",
+	"components.input-date-time.errorMaxDateOnly": "$$$$$$$ $$$$$ $$$$$ {maxDate} $$$$$$$$",
+	"components.input-date-time.errorMinDateOnly": "$$$$$$$ $$$$$ $$$ {minDate} $$$$$$$$",
+	"components.input-date-time.errorOutsideRange": "$$$$$$$ $$$$$ $$$$$$$$ {minDate} $$$ {maxDate} $$$$$$$$",
+	"components.input-date-time.time": "$$$$$",
+	"components.input-date.clear": "$$$$$$$$",
+	"components.input-date.errorMaxDateOnly": "$$$$$$$ $$$$$ $$$$$ {maxDate} $$$$$$$$",
+	"components.input-date.errorMinDateOnly": "$$$$$$$ $$$$$ $$$ {minDate} $$$$$$$$",
+	"components.input-date.errorOutsideRange": "$$$$$$$ $$$$$ $$$$$$$$ {minDate} $$$ {maxDate} $$$$$$$$",
+	"components.input-date.openInstructions": "$$$$$$$$$ $$$$$$$$$$$$$$$$ {format}. $$$$$$$$$ $$$$$ $$$$$$$$ $$$ $$$$$ $$$ $$$$$$$ $$$ $$$ $$$$$-$$$$$$$$ $$$ $$$$$$$$.",
+	"components.input-date.setToToday": "$$$$$$$$$$$$ $$$ $$$$$$$$$",
+	"components.input-search.clear": "$$$$$$$$$$$$$$$$ $$$$$$$$",
+	"components.input-search.search": "$$$$$$$$",
+	"components.input-time-range.endTime": "$$$$$$$$$$",
+	"components.input-time-range.errorBadInput": "{startLabel} $$$$$ $$$$$ {endLabel} $$$$$$$$",
+	"components.input-time-range.startTime": "$$$$$$$$$$$$",
+	"components.list-item-drag-handle.default": "$$$$$$$$$$$$ $$$$$ {name} $$$$$$$$$ $$$$$$$$$$$$$$$$",
+	"components.list-item-drag-handle.keyboard": "$$$$$$$ $$$$$$$$$ $$$$$$$$$$$$$$$$, $$$$$$$$$ $$$$$$$$$ {currentPosition} $$$$ {size}. $$$$ $ $$$$ $$$$$ $$$$$ $$$$$$$$$$$$$$, $$$$$$$ $ $$$ $$$ $$$$$ $$$$$$$$ $$$ $$$$$$$$.",
+	"components.menu-item-return.return": "$$$$$$$$$ $$$$$$$ $ $$$$$$$ $$$$$ $$$$ $$$$$$$$ $$$$$.",
+	"components.menu-item-return.returnCurrentlyShowing": "$$$$$$$$$ $$$$$$$ $ $$$$$$$ $$$$$ $$$$ $$$$$$$$ $$$$$. $ $$$$$$$$$ {menuTitle}.",
 	"components.meter-mixin.commaSeperatedAria": "{term1}, {term2}",
-	"components.meter-mixin.fraction": "{x}∕{y}",
-	"components.meter-mixin.progressIndicator": "Voortgangsindicator",
-	"components.more-less.less": "minder",
-	"components.more-less.more": "meer",
-	"components.switch.visibility": "Zichtbaarheid",
-	"components.tabs.next": "Naar voren scrollen",
-	"components.tabs.previous": "Naar achteren scrollen"
+	"components.meter-mixin.fraction": "{x}${y}",
```

Comment:
This is meant to show a fraction... not sure the `/` character should be translatable? 🤷